### PR TITLE
[SDS-NNN] Minor improvements

### PR DESCRIPTION
**Pull request by QFer**
* The rendering of readthedocs isn't always correct
*  Some spelling stuff

### CODE REVIEW

File: .readthedocs.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ build:
   image: latest
 
 python:
-  version: 3.8
+  version: 3.9
```
Comment (peendebak) [inline .readthedocs.yml]: RTD does not support 3.9
```suggestion
  version: 3.8
```

File: README.md [hunk 2]
```diff
@@ -64,10 +67,10 @@ or when you want to choose which example notebook to run from the browser do:
 jupyter notebook --notebook-dir="docs/notebooks"
 ```
 
-and select an ipython notebook (file with extension `ipynb`) to run from one of the directories.
+and select an IPython notebook (file with extension `ipynb`) to run from one of the directories.
```
Comment (peendebak) [inline README.md]:
```suggestion
and select a Jupyter notebook (file with extension `ipynb`) to run from one of the directories.
```
ipython has been replaces/renamed by Jupyter. This holds on more locations, so perhaps just rename it one by one

**Review by peendebak — DISMISSED** [review]

File: .readthedocs.yml (hunk 1 above)
Comment (QFer) [reply .readthedocs.yml]: changed

File: README.md (hunk 2 above)
Comment (QFer) [reply README.md]: Also changed all the instances

**Review by peendebak — APPROVED** [review]

**Comment by peendebak** [comment]
@QFer I approved the PR, but triggered the build again as the new api release might interfere with this PR.

**Pull request merged**